app/styles/_profile.scss

```diff
@@ -0,0 +1,147 @@
+@import "app-variables";
```

Comment:
Some of the styling in this file seems to be related only to the component. Maybe you could make a style sheet for the component in the `styles/components` directory? then you can import the sheet into app.scss with `@import "components/{{name-of-your-style-sheet-without-extension}}";`